indexdoc.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+   <meta charset="UTF-8">
+   <link rel="stylesheet" href="styles.css">
+   <title>Technical Documentation Page</title>
+ </head>
+
+ <body>
+   <main>  
+ <main id="main-doc" >JS Documentation
```

Comment:
hi @Eldonratzlaff , good job with the review :+1: , however please try to always follow the below points :point_down: 

- Remember to explain why you request a change - it might be obvious for you but not for someone else.
- Remember to be polite and clear in your comments - your job here is to help another person improve in a friendly way.
- [OPTIONAL] You can include emojis and gifs to make your code review more human-friendly alien.

for example we can say _**'I think it would be better to use an h1 tag here, as it is semantically more appropriate and will also increase the site's accessibility' you can also provide a [link to html semantics](https://developer.mozilla.org/en-US/docs/Glossary/Semantics)**_